javascript-job-search-mvp/index.html

```diff
@@ -75,7 +75,11 @@
 						<li><a href="https://vetswhocode.io/apply"><span class="link-name">Apply</span></a></li>
 						<li><a href="https://vetswhocode.io/contact"><span class="link-name">Contact</span></a></li>
 						<li><a href="https://vetswhocode.io/blog"><span class="link-name">Blog</span></a></li>
+<<<<<<< HEAD
```

Comment:
a little worried about these markers. I think they may have been pasted with your code when you redid your PR.

Reply:
```
// All variables for Submit Button 
const submitButton = document.getElementById("submit-button");
const zipCode = document.getElementById("zipCode");
const isRemote = document.getElementById("inlineFormCheck");
const range = document.getElementById("range");
const formData = {};
// Triggers the function when the search button is clicked
function submitButtonEvent (event) {
  event.preventDefault()
  formData.zipCode = zipCode.value;
  formData.isRemote = isRemote.checked;
  formData.range = range.value;
  console.log(formData);
}
submitButton.addEventListener("click", submitButtonEvent);
```
I think what went wrong is that you pasted the code from your last pull request to try to fix the merge conflict. You inadvertently pasted the markers it displayed in the last merge conflict. I think what I would do is:
1. rebase main on your machine
2. make a brand new branch with a different name than you used before
3. paste the code above in the javascript file
4. push the code and do another pull request

Either do that or go through your code on this branch on your computer and remove all the markers.